docs/sources/whatsnew/_index.md

```diff
@@ -1,8 +1,33 @@
 +++
-title = "What's new in Grafana"
+title = "What's new"
 aliases = ["/docs/grafana/latest/guides/"]
+weight = 1
 +++
 
-# What's new
+# What's new Grafana
 
-This section contains pages with Grafana release notes.
+Grafana is changing all the time. Here you can find articles with release highlights.
```

Comment:
Conciser alternative: "Grafana is changing all the time. Here are the release highlights."